eagle-core/eagle-alert-parent/eagle-alert/alert-common/src/main/java/org/apache/eagle/alert/engine/model/AlertStreamEvent.java

```diff
@@ -35,6 +35,7 @@
     private StreamDefinition schema;
     private String createdBy;
     private long createdTime;
+    private boolean isDuplicate = false;
```

Comment:
currently it is only for test. But should not "duplicate" be one property for StreamEvent? It sounds more naturally.